### PR TITLE
Better Spacing

### PR DESCRIPTION
**Pull request by nhubbard**
I improved the spacing for all the files.

### CODE REVIEW

**Comment by spiralx** [comment]
How is removing every blank line in the code better?

**Comment by nhubbard** [comment]
I (personally) was taught that the less blank lines there are, the better. But that doesn't mean that I'm right :smile:

**Comment by spiralx** [comment]
It just makes is really hard to see where the code is logically divided. Like removing paragraph breaks from text, it just makes it much harder to work out what's going on.

File: tests/test.mousetrap.js [hunk 1]
```diff
@@ -589,96 +447,71 @@ describe('Mousetrap.bind', function() {
...
         mousetrap.bind('a', spy);
-
         KeyEvent.simulate('a'.charCodeAt(0), 65);
-
         expect(spy.callCount).to.equal(1, 'callback should fire once');
         expect(spy.args[0][0]).to.be.an.instanceOf(Event, 'first argument should be Event');
         expect(spy.args[0][1]).to.equal('a', 'second argument should be key combo');
     });
-});
+});
```
Comment (kafeltz) [inline tests/test.mousetrap.js]: I like new ending lines

**Pull request closed without merging**